pages.fr/common/aapt.md

```diff
@@ -0,0 +1,17 @@
+# aapt
+
+> Android Asset Packaging Tool.
+> Compile et empaquette les ressources d'une application Android.
+> Plus d'informations : <https://elinux.org/Android_aapt>.
+
+- Liste les fichiers contenus une archive APK :
+
+`aapt list {{chemin/vers/app.apk}}`
+
+- Affiche les metadatas d'une application (version, permissions, etc.) :
```

Comment:
```suggestion
- Affiche les metadatas d'une application (version, autorisations, etc.):
```

Google translate suggests this translation. :)

Reply:
both are synonyms and work in that context ^^
So if you prefer `autorisation` no problem 😉